scripts/seed-reusable-workflow-registry.py

```
@@ -0,0 +1,110 @@
+import importlib.util
+import sys
+from datetime import date
+from pathlib import Path
+
+from ruamel.yaml import YAML
+
+_SELF_DIR = Path(__file__).resolve().parent
+_VALIDATOR_PATH = _SELF_DIR / "check-required-checks.py"
+
+
+def _load_validator():
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_load_validator`

(ANN202)